development/rollout-operator-role.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: rollout-operator-role
+  namespace: rollout-operator-development
+rules:
+  - apiGroups:
+      -
```

Comment:
why is this empty?

Reply:
An empty value means the "core" group (see https://kubernetes.io/docs/reference/access-authn-authz/rbac/#role-example for an example)

Reply:
`error: error validating "/Users/christopher.li/rollout-operator/development/rollout-operator-role.yaml": error validating data: ValidationError(Role.rules[0].apiGroups): unknown object type "nil" in Role.rules[0].apiGroups[0]; if you choose to ignore these errors, turn validation off with --validate=false` when I try to run ./apply.sh

Reply:
What version of Kubernetes and `kubectl` are you using?